Assets/Scripts/Gamemode/RacingGamemode.cs

```diff
@@ -44,12 +57,36 @@ public override string GetVictoryMessage(float timeUntilRestart, float totalServ
         {
             if (PhotonNetwork.offlineMode)
             {
-                var num = (((int)(totalServerTime * 10f)) * 0.1f) - 5f;
-                return $"{num}s !!\n Press {InputManager.GetKey(InputUi.Restart)} to Restart.\n\n\n";
+                if (string.IsNullOrEmpty(localRacingResult))
+                {
+                    //also subtract 1 delta time cos this will be called the first time the update next to the game win 
+                    totalServerTime -= (RacingGamemode.StartTimerCountdown-UnityEngine.Time.deltaTime);
+                    //the following one is to write the result in the hh:mm:ss,sss format  ((int) totalServerTime/3600).ToString("00") + ":"+((int)totalServerTime/60%60).ToString("00") + ":" +(totalServerTime%60).ToString("00.000");
+                    localRacingResult = totalServerTime.ToString("f2");
+                }
+                return $"{localRacingResult}s !!\n Press {InputManager.GetKey(InputUi.Restart)} to Restart.\n\n\n";
             }
             return $"{localRacingResult}\n\nGame Restart in {(int) timeUntilRestart}";
         }
 
+        /// <summary>
+        /// format the text that has to be printed on top of the screen for this specific gamemode
+        /// Is just temporary as things will have to be changed and handled locally without needing to go throught FGMKII core2
+        /// </summary>
+        /// <param name="time">it require the roundtime in csec (0.1sec) so it require you to pass the time multiplied by RacingGamemode.startTimerCountdown so it will hold also info about the first dec</param>
+        /// <param name="totalRoomTime"></param>
+        /// <returns></returns>
+        public override string GetGamemodeStatusTop(int time = 0, int totalRoomTime = 0)
+        {
+            time -= RacingGamemode.IntRoundTimeStatusTopScale*RacingGamemode.StartTimerCountdown;
+            if (time > 0)
+                //if the starting time has passed it return 
```

Comment:
Your if statement already says that if time is larger than 0, so useless comments (also for the else statement)

Reply:
well just to clarify as time gets an operation done before